### ISSUE TITLE
ScreenReader does not announces the column and row values for the Months "MAY ,JUNE ..." correctly.

### ISSUE THREAD

**Issue by caiomontei** [OfficeDev/office-ui-fabric-react#9641]
## Codepen demonstrating the issue
https://codepen.io/anon/pen/EBEvyg?&editable=true

## Actual behavior:
All months are being read as column one. (E.g. May is being announce as row 1 column 5)
## Expected behavior:
Months should be announced according to the table layout displayed in the screen (E.g. May should be announced as row 2 column 1)
## Documentation describing expected behavior
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

**Comment by khmakoto** [comment]
Hi @caiomontei! Thanks for filing this issue!

I gather that the issue relates to months being read as all of them being part of a single row instead of three, correct? I think this is indeed a bug and I'll try to have a fix as soon as I can.

**Comment by caiomontei** [comment]
Hi,

Yes, exactly that. Thanks!

**Comment by khmakoto** [comment]
I've taken a look at this and it seems like we'd need to restructure how we render the months in the `Calendar` for this to work correctly. We currently use a map of the months inside `buttons` with `role="gridcell"` set inside a single `div` with `role="row"` set. This causes the situation described above.

Because of the huge effort that this would require in terms of restructuring, I'll be putting this on backlog review for now. Thanks!

**Comment by lorejoh12** [comment]
This issue is fixed in the date-time package version of Calendar (which did that button refactoring described above). Closing, although not everyone will get the fix until we explicitly deprecate OUFR Calendar.
